package.json

```diff
@@ -176,6 +167,7 @@
     "ora": "5.4.1",
     "pacote": "20.0.0",
     "parse5-html-rewriting-stream": "7.0.0",
+    "patch-package": "^8.0.0",
```

Comment:
Can't we use `pnpm patch` instead of `patch-package`?

Reply:
We are in a hybrid state of yarn and pnpm, so we can't unfortunately